src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx

```diff
@@ -137,15 +138,31 @@ const ListingBody = React.memo(
     const listingItems =
       data?.query?.length > 0 ? querystringResults?.items || [] : folderItems;
 
-    const templateConfig = config.blocks.blocksConfig.listing.templates;
+    let ListingBodyTemplate;
+    let templateConfig;
 
-    let templateName =
-      data.template && !!templateConfig[data.template]
-        ? data.template
-        : 'default';
+    // Legacy support if template is present
+    if (data.template && !data.variation) {
+      const variations =
+        config.blocks?.blocksConfig['listing']?.variations || [];
```

Comment:
Non dovrebbe essere `templates`?
```suggestion
      const variations =
        config.blocks?.blocksConfig['listing']?.templates || [];
```

Reply:
no no è variations.. lo hanno chiamato cosi...

Reply:
okok capito